javascriptv3/example_code/sqs/src/sqs_getqueueurl.ts

```diff
@@ -27,11 +27,11 @@ const REGION = "REGION"; //e.g. "us-east-1"
 const params = { QueueName: "SQS_QUEUE_NAME" }; //SQS_QUEUE_NAME
 
 // Create SQS service object
-const sns = new SQSClient({ region: REGION });
+const sqs = new SQSClient({ region: REGION });
```

Comment:
"SQ" -> "Amazon Simple Queue Service (Amazon SQS)" on first mention in a code file.